README.md

```diff
@@ -1,37 +1,35 @@
 # Alacritty Theme
 
-[![license](https://img.shields.io/github/license/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/blob/master/LICENSE)
-[![GitHub tag](https://img.shields.io/github/tag/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/tags)
-[![GitHub last commit (branch)](https://img.shields.io/github/last-commit/eendroroy/alacritty-theme/master.svg)](https://github.com/eendroroy/alacritty-theme)
+Collection of colorschemes for easy configuration of the [Alacritty terminal
+emulator].
 
-[![GitHub issues](https://img.shields.io/github/issues/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/issues)
-[![GitHub closed issues](https://img.shields.io/github/issues-closed/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/issues?q=is%3Aissue+is%3Aclosed)
-[![GitHub pull requests](https://img.shields.io/github/issues-pr/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/pulls)
-[![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/pulls?q=is%3Apr+is%3Aclosed)
+[Alacritty terminal emulator]: https://github.com/alacritty/alacritty
 
-This repo lists (copied or slightly modified) color schemes for `alacritty`
+## Installation
 
-- collected from various sources
-- converted from other terminal emulator themes
-- created from vim or emacs color schemes.
+### Imports
 
-## How to apply?
+Clone the repository, or download the theme of your choice:
 
-There are multiple ways - 
+```sh
+# We use Alacritty's default Linux config directory as our storage location here.
+mkdir -p ~/.config/alacritty/themes
+git clone https://github.com/alacritty/alacritty-theme ~/.config/alacritty/themes
+```
 
-1. Import the scheme in `alacritty.yaml` file:
+Add an import to your `alacritty.yml` (Replace `{theme}` with your desired
+colorscheme):
 
-   Clone the repository:
+```yaml
+import:
+ - ~/.config/alacritty/themes/themes/{theme}.yaml
+```
 
-      `git clone https://github.com/eendroroy/alacritty-theme.git ~/.alacritty-colorscheme`
-   
-   And add the below line into `alacritty.yaml`:
-     ```yaml
-     import:
-       - ~/.alacritty-colorscheme/themes/{scheme_name}.yaml
-     ```
-2. Copy the content of `yaml` file in the `colors` section into `~/.config/alacritty/alacritty.yml` file.
-3. Use [alacritty-colorscheme](https://github.com/toggle-corp/alacritty-colorscheme). Read details here: [Docs](https://github.com/toggle-corp/alacritty-colorscheme/blob/master/README.md)
```

Comment:
It's possible this is a great tool to switch through colorschemes but I haven't had the chance to review its code and author yet. So I'd rather remove it for now so we're not advertising subpar stuff in an official manner.